std_msgs/Header header
string room_id
PersonDetection[] detections
